lessons/beginners/first-steps/index.md

```diff
@@ -1,50 +1,25 @@
-## První příkazy v Pythonu
+# Interaktivní režim Pythonu
 
-Pojďme si vyzkoušet, že nainstalovaný Python funguje!
+Chceš-li si začít hrát s Pythonem, otevři *příkazový řádek* a aktivuj virtuální prostředí.  Zkontroluj si, že na začátku příkazové řádky ti svítí `(venv)`.
 
-Zkontroluj si, že máš aktivované virtuální prostředí (na začátku příkazové
-řádky ti svítí `(venv)`).
-
-Je-li tomu tak, nezbývá než – konečně – pustit Python.
-K tomu použij příkaz `python`:
+Je-li tomu tak, nezbývá než – konečně – pustit Python. K tomu použij příkaz `python`:
 
 ``` console
-(venv)$ python
-Python 3.6.0 (default, Jan 26 2014, 18:15:05)
-[GCC 4.8.2 20131212 (Red Hat 4.8.2-7)] on linux
+$ python
+Python 3.6.6 (...)
 Type "help", "copyright", "credits" or "license" for more information.
+>>>
 ```
 
-Příkaz vypíše několik informací.
-Z prvního řádku se můžeš ujistit, že používáš Python 3.
-
-Python pak třemi „zobáčky“ `>>>` poprosí o instrukce.
-Je to jako v příkazové řádce, ale místo příkazů jako
-`cd` a `mkdir` sem budeš psát příkazy Pythonu.
-
-Nejjednodušší příkaz Pythonu je prosté číslo. Zkus to:
-
-```pycon
->>> 1
-1
->>> 42
-42
->>> -8.3    # (Python používá desetinnou tečku)
--8.3
-```
-
-> [note]
-> Zobáčky `>>>` i odpověď vypisuje sám Python!
```

Comment:
tohle odebíráš schválně?

Reply:
Ne, mělo by to tam zůstat. Díky za upozornění!